var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -86,7 +86,7 @@ class Seacas(CMakePackage):
     # Everything should be compiled position independent (-fpic)
 
     depends_on('netcdf-c@4.8.0:+mpi+parallel-netcdf', when='+mpi')
-    depends_on('netcdf-c@4.8.0:~mpi', when='~mpi')
+    depends_on('netcdf-c@4.8.0:~mpi~parallel-netcdf', when='~mpi')
```

Comment:
That is a good question and I think the answer is, "it is possible", because `parallel-netcdf`  does not depend on `netcdf-c`. However, the `netcdf-c` package recipe has the following:
```
 depends_on('mpi', when='+parallel-netcdf')
```
which makes the above dependency necessary in `seacas`. I was not inclined to mess with the `netcdf-c` packages in this PR.

Reply:
I reverted this change as the clingo concretizer handles this natively.